apps/codeforafrica/src/payload/collections/Posts.js

```diff
@@ -23,6 +23,7 @@ const Posts = {
     description: "Stories and Opportunities",
     group: "Publication",
     useAsTitle: "title",
+    listSearchableFields: ["content", "excerpt"],
```

Comment:
No need to include `title`?

Reply:
@kilemensi You can already search by title. I think it is made possible by `useAsTitle: "title"`